app-laptop/lphdisk/lphdisk-0.9.1-r3.ebuild

```diff
@@ -0,0 +1,29 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+inherit eutils toolchain-funcs
+
+DESCRIPTION="utility for preparing a hibernation partition for APM Suspend-To-Disk"
+HOMEPAGE="https://web.archive.org/web/20160422052247/http://www.procyon.com/~pda/lphdisk/"
+SRC_URI="http://www.procyon.com/~pda/lphdisk/${P}.tar.bz2"
+
+LICENSE="Artistic"
+SLOT="0"
+KEYWORDS="~x86"
+IUSE=""
+
+DEPEND="sys-libs/lrmi"
+
+src_prepare() {
+	epatch "${FILESDIR}"/${P}-gentoo.patch
```

Comment:
Nop. Please create a `PATCH` array and add this patch to it (outside this function).